src/config/index.js

```diff
@@ -57,15 +57,18 @@ export default {
       Wales: 'teamwales@quidditchuk.org',
       Scotland: 'teamscotland@quidditchuk.org',
     },
+    scoutingEmail: 'quk-scouting@gmail.com',
     templates: {
       welcome: 19455866,
       forgotPassword: 19133707,
       contactForm: 19443708,
       volunteerForm: 19447684,
       newMember: 19486834,
-      nationalTeam: 19535959,
       ediCommitteeForm: 20291343,
       youthCoachForm: 21146875,
+      nationalTeamInterest: 99,
+      scoutingApplication: 1234,
+      scoutingResponse: 5678,
```

Comment:
```suggestion
      nationalTeamInterest: 24631798,
      scoutingApplication: 24628586,
      scoutingResponse: 24632999,
```

National Team Interest (Admin)
<img width="1288" alt="Screenshot 2021-08-06 at 17 58 43" src="https://user-images.githubusercontent.com/10368230/128546512-155de0f5-39bc-42ca-a1f0-c9a943def112.png">

Scouting Application (Admin)
<img width="1287" alt="Screenshot 2021-08-06 at 17 59 01" src="https://user-images.githubusercontent.com/10368230/128546546-5fb4bb63-f644-46e8-9fb4-09efe8b869cb.png">

Scouting Application (User)
<img width="1165" alt="Screenshot 2021-08-06 at 18 00 57" src="https://user-images.githubusercontent.com/10368230/128546567-2f35cdfa-fffa-477c-9045-ee8644ea1281.png">